scalableminds/webknossos
# make it possible again to kick user out of organization team

### Mailable description of changes:
 - It is now possible again to kick a user out of the organization team

### URL of deployed dev instance (used for testing):
- https://organizationteamremovable.webknossos.xyz

### Steps to test:
- create another team
- create another user, remove Connectomics department in their team list, add them as User to second team
- give the second team access to a dataset
- they should see that dataset, but not the others, even if they have Connectomics department in their allowed teams list

### Issues:
- fixes #2800

------
